accessible_api/AccessibilityRequestHandler.py

```diff
@@ -0,0 +1,124 @@
+""" Service requests for accessible objects """
+
+import sys
+from BaseHTTPServer import BaseHTTPRequestHandler
+from re import search
+from urlparse import urlsplit, parse_qsl
+from json import dumps
+from comtypes import CoInitialize
+from accessible_lib.scripts.accessible import accessible
+from accessible_lib.scripts.event import event
+from accessible_lib.scripts.commands import execute_command
+
+
+class AccessibilityRequestHandler(BaseHTTPRequestHandler):
+    def _successful_response(self, json):
+        self.send_response(200)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+        self.wfile.write(json)
+
+    def _bad_response(self, message):
+        self.send_response(400, message)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+    def _set_identifiers(self, params):
+        name = params.get('name')
+        role = params.get('role')
+
+        identifiers = {}
+        if name is not None:
+            identifiers["Name"] = name
+        if role is not None:
+            identifiers["Role"] = role
+
+        return identifiers
+
+    def do_GET(self):
```

Comment:
In the functions themselves, it is hard to follow all the query parsing stuff, I think you should just get things out of the query as you need them (I think urlparse parses it all into a dictionary for you). The identifiers function should go away too, it makes it hard to follow why you need keys that are capitalized somewhere down the line in some other object. If you ever need to do that, it should be done at a location where it is needed.